tests/e2e/performance/test_pvc_attachtime.py

```diff
@@ -157,26 +198,46 @@ def test_pod_start_time(self, pod):
         self.full_results = self.init_full_results(
             ResultsAnalyse(self.uuid, self.crd_data, self.full_log_path)
         )
+        self.full_results.add_key("storageclass", self.sc)
 
-        start_time = start_time_dict["web-server"]
-        logging.info(f"pod start time: {start_time} seconds")
-        if start_time > 30:
-            raise ex.PerformanceException(
-                f"pod start time is {start_time}," f"which is greater than 30 seconds"
+        time_measures = [t["web-server"] for t in start_time_dict_list]
+        for index, start_time in enumerate(time_measures):
+            # start_time = start_time_dict["web-server"]
+            logging.info(
+                f"{self.msg_prefix} pod number {index} start time: {start_time} seconds"
             )
-        self.full_results.add_key("storageclass", self.sc)
-        self.full_results.add_key("attach_time", start_time)
+            if start_time > 30:
+                raise ex.PerformanceException(
+                    f"{self.msg_prefix} Pod number {index} start time is {start_time},"
+                    f"which is greater than 30 seconds"
+                )
+
+        average = statistics.mean(time_measures)
+        logging.info(
+            f"{self.msg_prefix} The average time for the sampled {len(time_measures)} pods is {average} seconds."
+        )
+        self.full_results.add_key("attach_time_average", average)
+
```

Comment:
pleas add to the ES results also the results for all samples
self.full_results.all_results["attach_time"] = time_measures`

Reply:
Will do, thank you Avi.